lib/anycablebility/client.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'thread'
+require 'json'
+require 'websocket-eventmachine-client'
+
+require 'anycablebility/waiter'
+
+module Anycablebility
+  # This is a simple ActionCable client created for testing purposes
+  class Client
+    attr_reader :state
+
+    def initialize(cable_url, debug = false, ignore_message_types = [])
```

Comment:
I don't really understand what's the benefit of using Anyway for such task. I mean who will want to use Anycablebility inside a Rails app or configure it via environment variables?

I'm also not sure why should we introduce global state here instead of just passing config around. This makes no sense to represent our components as objects if they will rely on some global state, isn't it?